src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -925,6 +925,51 @@ have to be updated for changes to the rules above, or to support more deeply nes
 	border-color: var(--vscode-notificationsWarningIcon-foreground);
 }
 
+/**
+ * Styles for the `prompt instructions` attachment widget.
+ */
+.chat-attached-context .chat-prompt-instructions-attachments {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+	gap: 2px;
+}
+.chat-attached-context .chat-prompt-instructions-attachment {
+	display: flex;
+	gap: 4px;
+	border-color: currentColor;
+}
+.chat-attached-context .chat-prompt-instructions-attachment .codicon {
+	color: inherit;
+	text-decoration: none;
+}
+.chat-attached-context .chat-prompt-instructions-attachment .chat-implicit-hint {
+	opacity: 0.7;
```

Comment:
If this is meant to look like the implicit widget, would it make sense to share the styles?

Reply:
Yeah shared styles is what I had in the beginning but then decided that having separate ones is safer, so CSS changes a bit more isolated between components. Ideally we would have a set of common base styles shared between all similar components, but I didn't dare to do that refactoring in this PR reduce the PR scope 🤗